### ISSUE TITLE
Random multiple TTS's playing sequentially

### ISSUE THREAD

**Issue by JsChiSurf** [jishi/node-sonos-http-api#513]
Don't know if anyone else is seeing this, and I've yet to find a consistent pattern as to how / why this is happening, but from time to time, I will be kicking off an automated TTS to a single speaker (sometimes this is an actual TTS playing as a WAV file, other times a static MP3 clip), and before the requested TTS completes (ie, it will get cut off), or as soon as, another (perhaps default) TTS will play, for which no additional call / request has been made.

As mentioned, this doesn't happen all the time, maybe 1 in 6 or 8 requests, but I'm not sure what is going on / how this is happening.

**Comment by jishi** [comment]
I think what you are experiencing is the fact that multiple `say` commands would remember the initial state and restore it even if multiple `say` or `clip` commands would be issued. However, it would not remember it for intermixed `say/sayall` calls, because they are unaware of the other actions doing. 

Could this be the case?

**Comment by JsChiSurf** [comment]
So, I am not usually (if ever), making multiple say commands one after another.  I am certainly issuing multiple say commands throughout the day to the same speaker.

I switched from using the sayall command, to just "say" to a specific speaker, in order to increase the speed in which the say command is executed, since my PHP code continues an runs "other" tasks.  I actually wrote an async HTTP function so that I could effectively background the "say" call (I don't care about the return status), so that my code can continue on with its remaining functionality.

Are you saying by virtue of "remembering" the initial state and restoring, that it therefore plays the initial state audio upon completion of the requested TTS somehow?

Just to reiterate though, I am not issuing multiple say commands when this occurs.  Just one, but that one command is sometimes truncated with another click (initial state?) playing either over top if it, or right after.  Hope that makes sense.
